frontend/app/routes/$lang/_public/apply/$id/adult-child/children/$childId/information.tsx

```diff
@@ -182,13 +182,19 @@ export async function action({ context: { session }, params, request }: ActionFu
     });
   }
 
+  const ageCategory = getAgeCategoryFromDateString(parsedDataResult.data.dateOfBirth);
```

Comment:
Kind of a nitpick but this identical function call to `getAgeCategoryFromDateString(..)` occurs below on line 210. Consider refactoring to have a single function call and reusing the `ageCategory` local variable throughout.

This same comment can be applied to the `child` flow's `information.tsx`.

Reply:
good catch